site/content/docs/5.2/migration.md

```diff
@@ -9,6 +9,204 @@ aliases:
 toc: true
 ---
 
+## v5.2.1
+
+<hr class="mb-4">
+
+Boosted v5.2.1 is here with fixes from our latest minor release, v5.2. These changes include bug fixes, documentation updates, and some dependency updates, but also a new Tags component and a lot of new dark variants for our components.
+
+If you need more details about the changes, please refer to the [v5.2.1 release](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/tag/v5.2.0).
+
+### Components
+
+- **Back to top**
+  - <span class="badge bg-danger">Breaking</span> Back to top 'Label inside' variant was removed because not compliant with Orange Design System. Even if the rendering could still work, it is recommended to only use the versions presented in the documentation.
+
+- **Breadcrumb**
+  - <span class="badge bg-success">New</span> Breadcrumb now has a dark variant.
+
+- **Footers**
+  - <span class="badge bg-warning">Warning</span> All Footers examples have been modified to use the "Terms and conditions" wording instead of "Terms & Conditions". Please reflect this modification into your websites.
+
+- **List group**
+  - <span class="badge bg-success">New</span> List group now has a dark variant.
+
+- **Pagination**
+  - <span class="badge bg-success">New</span> Pagination now has a dark variant.
+
+- **Scrollspy**
+  - <span class="badge bg-success">New</span> Scrollspy has a new `data-bs-threshold` data attribute.
+
+- **Stepped process**
+  - <span class="badge bg-success">New</span> Stepped process now has a dark variant.
+
+- **Tags**
+  - <span class="badge bg-success">New</span> Tags now have a dark variant.
+
+- **Toasts**
+  - <span class="badge bg-warning">Warning</span> Changed the markup for Toasts with a custom content (toast message and close button). Please reflect this modification into your websites.
+    <details>
+      <summary>More info</summary>
+      {{< markdown >}}
+```diff
+<div class="toast align-items-center" role="alert" aria-live="assertive" aria-atomic="true">
+  <div class="d-flex">
+-    <div class="toast-body">
++    <div class="toast-body my-auto">
+      Hello, world! This is a toast message.
+    </div>
+-    <button type="button" class="btn-close me-2 m-auto" data-bs-dismiss="toast"><span class="visually-hidden">Close</span></button>
++    <button type="button" class="btn-close ms-auto" data-bs-dismiss="toast"><span class="visually-hidden">Close</span></button>
+  </div>
+</div>
+```
+      {{< /markdown >}}
+    </details>
+
+- **Tooltips**
+  - <span class="badge bg-warning">Warning</span> Tooltips examples applied on SVGs have been updated to use `focusable="false"` and `aria-hidden="true"` because SVGs do not carry any specific information so they should not be readable by screen readers. Please reflect this modification in your websites.
```

Comment:
Re reading this, I wonder if this change should be in the Tooltip section or inside the contents section without the mention of tooltips ?